### PR TITLE
feat: 5.1.0 references

### PR DESCRIPTION
**Pull request by MillenniumFalconMechanic**
## Reason for Change

- #6877 

## Changes

- Updated schema references from 5.0.0 to 5.1.1 (excluding apps-specific constants; see [PR](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7078))

## Testing steps

- Tested on local.

## Checklist 🛎️

- [ ] Add product, design, and eng as reviewers for rdev review

### CODE REVIEW

File: backend/cellguide/pipeline/constants.py [hunk 1]
```diff
@@ -15,4 +15,4 @@
 
 CELL_GUIDE_DATA_BUCKET_PATH_PREFIX = "s3://cellguide-data-public-"
 
-CELL_GUIDE_PINNED_SCHEMA_VERSION = "5.0.0"
+CELL_GUIDE_PINNED_SCHEMA_VERSION = "5.1.0"
```
Comment (nayib-jose-gloria) [inline backend/cellguide/pipeline/constants.py:18]: I remember you asking about this previously--after some more thought I think we should leave it at 5.0.0 and let Apps team handle when to bump it. 

We do want them to update as soon as possible but there may be a reason they want to delay

File: backend/wmg/data/constants.py [hunk 2]
```diff
@@ -14,4 +14,4 @@
     "EFO:0010010": "CEL-seq2",
 }
 
-WMG_PINNED_SCHEMA_VERSION = "5.0.0"
+WMG_PINNED_SCHEMA_VERSION = "5.1.0"
```
Comment (nayib-jose-gloria) [inline backend/wmg/data/constants.py:17]: Same as Cell Guide pinned version--after some more thought I think we should leave it at 5.0.0 and let Apps team handle when to bump it.

**Review by nayib-jose-gloria — COMMENTED** [review]
open thought, not a new requirement: I wonder if we should consider maintaining a "latest" branch / tag in single-cell-curation that points to the latest schema reference or something rather than having to bump these links every time. Not a huge priority but if we have time to think on that cc: @Bento007

**Comment by Bento007** [comment]
refresh bot

**Comment by Bento007** [comment]
refresh PR

**Review by nayib-jose-gloria — APPROVED** [review]

**Pull request merged**